src/translations/fr.json

```
@@ -1061,8 +1056,8 @@
     "cancelCutNetworkModificationSubtree": "Annuler couper le sous arbre",
     "removeNetworkModificationSubtree": "Supprimer le sous arbre",
     "restoreNodes": "Restaurer des nœuds",
-    "restoreNodesText": "Veuillez sélectionner les nœuds à restaurer",
-    "deleteNodesText": "Êtes-vous sûr(e) de vouloir supprimer définitivement {numberToDelete} nœuds ou sous-arbres ?",
+    "restoreNodesText": "Veuillez sélectionner les nœuds à restaurer :",
```

use non-breaking space in french
```suggestion
    "restoreNodesText": "Veuillez sélectionner les nœuds à restaurer\u0a00:",
```